src/mixins/common-mixin.ts

```diff
@@ -37,6 +38,24 @@ export default class CommonMixin extends Vue {
     })
   }
 
+  /**
+   * Identifies the first invalid flag and scrolls to the component
+   * @param validatorFlags The component valid flags
+   */
+  validateAndScroll (validatorFlags: ValidFlagsIF): boolean {
+    // Create an array of the flag values
+    const validFlagArray = Object.keys(validatorFlags).map(key => validatorFlags[key])
```

Comment:
This should also work:
`const validFlagArray = [ ...validatorFlags ]`

Reply:
Cool, i'll give it a try.

Reply:
I get: 
`TypeError: Invalid attempt to spread non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.`

The object being non-iterable. 
We can spread an object but not sure how/if it can be spread into an array.

Reply:
Oh well.

(I saw this work recently when the business snapshot array was spread into an object. But maybe it's just that TS didn't catch it.)